ddp-workspace/projects/ddp-sdk/src/lib/services/internationalization/languageService.service.ts

```diff
@@ -67,4 +72,11 @@ export class LanguageService {
     const obs = this.changeLanguageObservable(languageCode);
     return obs !== null;
   }
+
+  public getAppLanguageCode(): string {
```

Comment:
This method is useful for the `translateFactory` to init the application with appropriate language.